tests/Resources/DataFixtures/Webspace/multiple/sulu.io.xml

```diff
@@ -36,7 +40,7 @@
     </navigation>
 
     <resource-locator>
-        <strategy>short</strategy>
```

Comment:
Is that a real strategy? If so, then I couldn't find any classes implementing the strategy.